crypto/ec/ec_asn1.c

```diff
@@ -747,7 +747,7 @@ EC_GROUP *EC_GROUP_new_from_ecparameters(const ECPARAMETERS *params)
     if (params->cofactor == NULL) {
         BN_free(b);
         b = NULL;
-    } else if ((b = ASN1_INTEGER_to_BN(params->cofactor, b)) == NULL) {
+    } else if (ASN1_INTEGER_to_BN(params->cofactor, b) == NULL) {
```

Comment:
Do we have the same bug on line 733 where we do much the same handling for `a`

Reply:
Reusing the values of a & b to store order and cofactor was probably not a great choice to begin with.